contracts/Ethleverage.sol

```diff
@@ -225,6 +249,25 @@ contract Ethleverage {
     return (investor.layers, investor.prinContr, investor.CR, investor.cdps, investor.daiAmountFinal, investor.ethAmountFinal);
   }
 
+  // For testing purposes only!!!
+  function getVariables() constant public
```

Comment:
If we declare the contract addresses as public vars we don't need this. We can also just have getters for each of them, might be cleaner?

Reply:
Definitely, please revise where you see fit. This test function was just used to grab everything at once.